data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1202 @@
+				goto questions
+			label space
+			`	"Very little. The pirates wouldn't let us leave the planet, and jump drives are precious to them. They cannot risk drives on science expeditions. Still, we have managed a few trips."`
+			`	"There is no evidence that any civilization has ever reached this region of space until us. There are many space-faring indigenous creatures that live in gas giants, and sometimes venture beyond. Most are harmless, so long as you leave them alone. There are flocking apex predators who hunt anything that moves. We suspect there is a north route from here to another galaxy, though we cannot find the starlanes yet."`
+			# The starlanes are not concealed. "We cannot find the starlanes" is a
+			# fourth wall breaking joke that refers to that part of the game still
+			# being under construction. That last sentence will be changed once the
+			# SMC is created.
+			`	"The planet below is a mystery to us. It has a thousand times the mass than it should and its gravitational field reads as such from a great distance. This tremendous gravitational field sustains the large asteroid belt. Somehow though, despite its incredible mass, as you approach the surface the field does not increase at anywhere near the rate that it should, and so when you stand upon it the gravity is only similar to that of Earth. There are many forms of radiation and strange materials inside the planet's caves. Perhaps, now that we are not developing pirate weapons, we can focus on exploration."`
+				goto questions
+			label leave
+			`	As you return to your ship, Teeneep interrupts you. "I need you to escort me one last time, Captain <last>. I must return to <destination> to resolve things with now that the threat is ended. You will be well-paid, of course."`
+			`	Anticipating your next question, she says, "The ships of the True Hai will be escorted back with us since the Philosophers are convinced they are at no risk, and still have a ship at their disposal. Some of the True Hai have chosen to remain here and join the philosophers. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. We will have to transport the surviving operatives back to Foundry, the Republic should be able to look after them again from there."`
+				accept
+	on stopover
+		dialog "The operatives depart your ship into the arms of the human teams waiting there to see to their health and collect the dead."
+	on complete
+		payment 1000000
+		fail "Hai Rescue [B01A] Permanent Fleets"
+	npc save accompany
+		government "Hai (Friendly Unfettered)"
+		personality escort heroic
+		ship "Shield Beetle (Sunbeam Korath Jump)" "Unfettered Flagship"
+	npc
+		government "Hai (Friendly Unfettered)"
+		personality escort heroic
+		fleet 3
+			variant
+				"Shield Beetle (Particle Jump)"
+		fleet 3
+			variant
+				"Shield Beetle (Plasma Blaster Jump)"
```

Comment:
What are the odds that >4 out of 10 ships were destroyed? I sort of picked a number I thought was fairly reasonable, figuring a couple were destroyed, and if some Hai stayed behind they might not have been able to bring every ship back with them.

Reply:
If the player is starting this before they left Hai space, then it is likely they'll rely entirely on the Shield Beetles for combat. I tuned the battle so the Shield Beetles are unlikely to win on their own.

Reply:
An earlier mission will have forced the player to have escorts, if only briefly, That's meant to be a pretty big hint... is there a way we can track (without making a mission for every ship) how many were destroyed? Or should I just revisit this and see what can be changed? It seems like it would cause a lot of political trouble if you just left a bunch of unfettered ships with jump drives there unsupervised and told the republic they'd 'find their own way home' unescorted.

Reply:
I have a few possible solutions in mind. I'm still working my way through bugs in part 1. I'll try my idea when I get to that point. Hopefully in another 10 hours or so.